source/Interface.h

```diff
@@ -198,6 +200,7 @@ class Interface {
 	private:
 		std::string name;
 		const Color *color = nullptr;
+		double mapColor = numeric_limits<double>::quiet_NaN();
```

Comment:
You need `<limits>` for this.

Reply:
Ah I see. I wonder why it worked fine for me. (the Mac CI build also worked so seems to be OS-related)